thread/src/components/AddContactForm.js

```diff
@@ -36,18 +46,19 @@ class AddContactForm extends Component {
 
     render() {
         return (
-            <Container style={{ backgroundColor: '#fff' }}>
+            <Container style={{ backgroundColor: '#8bc34a' }}>
```

Comment:
hardcoding colours not good, create a common component called `CommonContainer`

```
export default CommonContainer = styled(Container)`
  backgroundColor: '#8bc34a'
`
```
It may be a small file, however in the future when we need to change the background colour, we can update one file

Reply:
Sorry forgot to say, that I have seen the same styling in another PR